torch/nn/qat/modules/normalization.py

```diff
@@ -182,10 +182,10 @@ def from_float(cls, mod, qconfig=None):
 
 class LayerNorm(nn.LayerNorm):
     r"""
-    A LayerNorm module attached with FakeQuantize modules for both output
-    activation and weight, used for quantization aware training.
+    A LayerNorm module attached with FakeQuantize modules for output
+    activation, used for quantization aware training.
 
-    Similar to `torch.nn.LayerNorm`, with FakeQuantize modules initialized to
+    Similar to :class:`torch.nn.LayerNorm`, with FakeQuantize modules initialized to
```

Comment:
I think we do not need this class. By default, we insert an activation post process after the operation, without needing a new class.